cypress/integration/spec/part2.js

```diff
@@ -0,0 +1,46 @@
+import {commonActions} from "../../support/page_objects/part2/commonActions";
+import {productPage} from "../../support/page_objects/part2/productPage";
+import {loginPage} from "../../support/page_objects/part2/loginPage";
+import {topNavigation} from "../../support/page_objects/part2/topNavigation";
+import {addressPage} from "../../support/page_objects/part2/addressPage";
+import {insertAddressPage} from "../../support/page_objects/part2/insertAddressPage";
+
+describe('Part 2', () => {
+
+    beforeEach(() => {
+        cy.reload()
+    })
+
+    it('1', () => {
+        // Napisz test dodający koszulkę oraz buty do koszyka (bez użycia wyszukiwarki), następnie przy pomocy
+        // wyszukiwarki dodaj jakiś kosmetyk i doprowadź zamówienie do finalizacji :) - sprawdź czy wszystko poszło OK
+
+        // próbuję dużego stopnia abstrakcji
+        commonActions.addProductToCart(123)
```

Comment:
magic number
https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

Reply:
Teraz dane do tego testu są pobierane z JSONa.